bip-0039/german.txt

```diff
@@ -0,0 +1,2048 @@
+BEIWERK
+BELANGE
+BELEG
+BELIEBEN
+BEMALUNG
+BENEHMEN
+BENGEL
+BENOTUNG
+BENUTZER
+BENZIN
+BERATER
+BEREICH
+BERG
+BERICHT
+BERUF
+BESATZER
+BESCHEID
+BESITZ
+BESTECK
+BESUCH
+BETONUNG
+BETRIEB
+BETTRUHE
+BEUTEL
+BEWEIS
+BEWOHNER
+BEZIRK
+BEZUG
+BIATHLON
+BIEGUNG
+BIER
+BIEST
+BIKINI
+BILANZ
+BILD
+BINDUNG
+BIOGAS
+BIOLOGE
+BIOMASSE
+BIOTOP
+BIRKHAHN
+BIRNE
+BISTRO
+BITTE
+BITUMEN
+BIWAK
+BIZEPS
+BLASE
+BLATT
+BLAUWAL
+BLEICHE
+BLESSUR
+BLICK
+BLINKER
+BLITZ
+BLOCKADE
+BLUME
+BODEN
+BOHNE
+BOHRER
+BOILER
+BOJE
+BOLLWERK
+BOLZEN
+BOMBE
+BONSAI
+BOOT
+BORD
+BORSTE
+BOSHEIT
+BOTTICH
+BOYKOTT
+BRANDUNG
+BRATEN
+BRAUEREI
+BREMSE
+BRENNGUT
+BRETT
+BREZE
+BRIKETT
+BRILLE
+BRISANZ
+BROTLAIB
+BRUCH
+BRUNNEN
+BRUST
+BRUTZEIT
+BUBE
+BUCHSE
+BUCKEL
+BUG
+BUMERANG
+BUSFAHRT
+BUSREISE
+BUSSARD
+BUTAN
+CABRIO
+CELLO
+CELSIUS
+CENT
+CHEMIKER
+CHILI
+CHOR
+CHROM
+COCKTAIL
+COUSINE
+CURRY
+DACH
+DAHEIM
+DAMENRAD
+DAMPF
+DANK
+DARLEHEN
+DATEI
+DAUERTON
+DAUNE
+DEBATTE
+DECKEL
+DEFEKT
+DEFIZIT
+DEGEN
+DEICHSEL
+DEKADE
+DEKOR
+DELFIN
+DELIKT
+DELLE
+DEMENTI
+DENKMAL
+DEPONIE
+DESASTER
+DESIGNER
+DETEKTOR
+DEUTUNG
+DEVISE
+DEZEMBER
+DIAGNOSE
+DIALOG
+DICKICHT
+DIDAKTIK
+DIELE
+DIENST
+DIKTAT
+DILL
+DIODE
+DIOPTRIE
+DIPLOMAT
+DIREKTOR
+DIRIGENT
+DISKURS
+DISTEL
+DIVISION
+DOCHT
+DOGGE
+DOHLE
+DOLCH
+DOLDE
+DOLLAR
+DOMIZIL
+DOMPTEUR
+DORADE
+DORF
+DORSCH
+DOSIERER
+DOZENT
+DRACHE
+DRAHT
+DRALL
+DRANG
+DRECK
+DREHBUCH
+DREIECK
+DRESSUR
+DROHUNG
+DROMEDAR
+DRUCKER
+DUELL
+DUETT
+DUFT
+DUNG
+DUPLIKAT
+DURST
+DUSCHE
+DUTZEND
+DYNAMIK
+EBBE
+EBENE
+EBER
+ECHOLOT
+ECHTZEIT
+ECKBALL
+ECKE
+EDAMER
+EDELGAS
+EDITION
+EFEU
+EFFEKT
+EGERLING
+EGO
+EHEBETT
+EHEPAAR
+EHRE
+EHRGEIZ
+EIDECHSE
+EIGELB
+EIGNUNG
+EILTEMPO
+EIMER
+EINBRUCH
+EINDRUCK
+EINFUHR
+EINGRIFF
+EINHORN
+EINIGUNG
+EINKAUF
+EINLAGE
+EINNAHME
+EINRAD
+EINSATZ
+EINTOPF
+EINWAND
+EINZAHL
+EISDIELE
+EISEN
+EISFACH
+EISKANAL
+EISTANZ
+EISWEIN
+EISZEIT
+EIZELLE
+EKLAT
+EKSTASE
+ELAN
+ELCH
+ELEFANT
+ELEGANZ
+ELEKTRIK
+ELFMETER
+ELLBOGEN
+ELOQUENZ
+ELSTER
+ELTERN
+EMBLEM
+EMBRYO
+EMISSION
+EMPATHIE
+EMPFANG
+EMPORE
+EMULSION
+ENDLAGER
+ENDOSKOP
+ENDPHASE
+ENDSPURT
+ENERGIE
+ENGPASS
+ENKLAVE
+ENTGELT
+ENTROPIE
+ENTWURF
+ENZIAN
+ENZYM
+EPOCHE
+ERBAUUNG
+ERBFOLGE
+ERBGUT
+ERBPACHT
+ERBSE
+ERDAPFEL
+ERDBODEN
+ERDGAS
+ERDKUGEL
+ERDNUSS
+ERDREICH
+ERDTEIL
+ERDUNG
+ERFINDER
+ERFOLG
+ERGEBNIS
+ERHEBUNG
+ERLASS
+ERLE
+ERNST
+ERNTE
+EROBERER
+ERPEL
+ERREGER
+ERSATZ
+ERTRAG
+ERUPTION
+ERZ
+ESCHE
+ESELSOHR
+ESKAPADE
+ESKORTE
+ESSENZ
+ESSIG
+ESSTISCH
+ESTRICH
+ETAGE
+ETAPPE
+ETAT
+ETHIK
+ETIKETT
+ETUI
+EUTER
+EVIDENZ
+EWIGKEIT
+EXEMPEL
+EXIL
+EXISTENZ
+EXKURS
+EXOT
+EXPANDER
+EXPERTE
+EXPONAT
+EXZESS
+FABRIK
+FACHWERK
+FACKEL
+FADEN
+FAGOTT
+FAHNE
+FAHRRAD
+FAKIR
+FAKTOR
+FALKE
+FALLOBST
+FAMILIE
+FANFARE
+FANPOST
+FANTASIE
+FARBTON
+FASAN
+FASCHING
+FASSADE
+FAULHEIT
+FAUST
+FAZIT
+FEBRUAR
+FEDERUNG
+FEHLER
+FEIERTAG
+FEIGHEIT
+FEILE
+FEIND
+FELDWEG
+FELGE
+FELL
+FELSWAND
+FENCHEL
+FENSTER
+FERIEN
+FERKEL
+FESTUNG
+FETT
+FETUS
```

Comment:
Listed as ["Fötus"](https://de.wikipedia.org/wiki/F%C3%B6tus) in Wikipedia, feels more commonly used like that, but we can't use that spelling due to the Umlaut. Both spellings seems to be fine though, might be hard for a user though if he selectes "o" as a second character expecting to put in "foetus" and doesn't get a matching word. Replace with another word?

Reply:
word replaced